docs/dev/developer-guides/transactions/transactions.md

```diff
@@ -0,0 +1,62 @@
+# Overview
+
+Transactions in Ethereum are cryptographically signed instructions by an externally owned account (an account owned by a user and not by code). These instructions are stored in the blockchain and added to a block.
+
+The state of the Ethereum virtual machine (EVM) changes when a transaction is initiated. A transaction can be anything from sending ether to another account to invoking the functions of a smart contract.
+
+
+## Prerequisite
+
+We recommend you first read [accounts](https://ethereum.org/en/developers/docs/accounts/) to understand this page.
+
+## How a transaction works
+
+When a user initiates a transaction on Ethereum, some specific data is created;
+
+- Receiver: The recipient is the account's address to receive the transaction. The receiver can be a contract account or an externally owned account. Each transaction is aimed toward a specific recipient.
+
+- Nonce: This field displays the most recent transaction based on the account's counter, which maintains track of how many transactions it does. The network uses the transaction nonce to ensure that transactions are completed in the correct sequence.
+
+- Gas Price: Most transactions necessitate the payment of a fee to the transaction's author. This cost is computed per unit of gas. The unit is Wei, a smaller ether unit.
+
+- Gas Limit: The transaction author specifies the number of gas units used for the transaction. This is the total amount of gas that could be consumed.
+
+- Value: The quantity of Wei or Ether that the sender account wishes to transmit to the recipient is represented by the value.
+
+- Data: If the transaction receiver is a smart contract, the data contains information for the contract's functions to be executed. This comprises data with varying lengths.
+
+- Signature: A signature indicates who sent the communication. The signature is created when an externally owned account confirms and signs the transaction with its private key.
+
+### Transaction Types
+
+There are several kinds of transactions on Ethereum:
+
+- Simple or asset transfers: This refers to the regular tokens transfer in the form of ether from one account to another.
+
+- Contract deployment transactions: A contract deployment transaction occurs when a contract is deployed on the Ethereum network. Although this transaction contains the smart contract's bytecode as data, it lacks a receiver (a "to" address).
```

Comment:
Note that this is not the case on zkSync. 

https://v2-docs.zksync.io/dev/zksync-v2/contracts.html